src/fs/realNodeModulesDir.ts

```diff
@@ -0,0 +1,20 @@
+import fs = require('mz/fs')
+import path = require('path')
+
+const CACHE: {[prefix: string]: string} = {}
+
+export default async function realNodeModulesDir (prefix: string): Promise<string> {
```

Comment:
would it be possible to make this function stateless? Maybe the realNodeModules could be passed along with prefix in opts/ctx